packages/cli/src/lib/wrangler-config.ts

```diff
@@ -1,7 +1,7 @@
 export function getWranglerConfig(projectUuid: string, kvNamespaceId: string) {
   return {
     $schema: 'node_modules/wrangler/config-schema.json',
-    main: '.open-next/worker.js',
+    main: '../.open-next/worker.js',
```

Comment:
🍹 Def doesn't have to be figured out to merge this PR, but I think a really useful automated test in the future would test that this path always correctly points to where `.open-next` is.